### PR TITLE
fix(stepper): step title max width without description corner case

### PR DESCRIPTION
**Pull request by valentin-mladenov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?
Nested steppers will be updated by the parent stepper.

Issue Number: CDE-3000

## What is the new behavior?
Nested steppers will not be updated by the parent stepper.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by valentin-mladenov** [comment]
Fixing feedback from #2691

**Review by idonchev-vmw — DISMISSED** [review]

File: projects/angular/stepper/_stepper.clarity.scss [hunk 1]
```diff
@@ -76,7 +76,9 @@
     display: block;
     margin-bottom: density.$clr-base-vertical-offset-xl;
 
-    &:has(.clr-stepper-description) .clr-stepper-header > .clr-stepper-header-button > .clr-step-title-wrapper {
+    $directChildren: '> .clr-stepper-panel > .clr-stepper-inner-wrapper > .clr-stepper-header > .clr-stepper-header-button';
+
+    &:has(#{$directChildren} .clr-stepper-description) #{$directChildren} > .clr-step-title-wrapper {
```
Comment (dtsanevmw) [inline projects/angular/stepper/_stepper.clarity.scss]: My only suggestion is to drop the new `clr-stepper-inner-wrapper` and replace it with either `div` or `*` as we know what we have there and there is not much need to introduce new class just for applying the selector here, we can already do it safe enough.
```suggestion
    $directChildren: '> .clr-stepper-panel > * || div > .clr-stepper-header > .clr-stepper-header-button';

    &:has(#{$directChildren} .clr-stepper-description) #{$directChildren} > .clr-step-title-wrapper {
```

Comment (valentin-mladenov) [reply projects/angular/stepper/_stepper.clarity.scss]: Replcing it with a `*` and a comment.

```
    // `> *` will select all variations of that element.It's used as shorth cut for classes:
    // clr-stepper-panel-inactive, clr-stepper-panel-error and clr-stepper-panel-complete
    $directChildren: '> .clr-stepper-panel > * > .clr-stepper-header > .clr-stepper-header-button';
```

Comment (valentin-mladenov) [reply projects/angular/stepper/_stepper.clarity.scss]: updated in commit 3d6b6ea.

**Review by idonchev-vmw — DISMISSED** [review]

**Review by dtsanevmw — APPROVED** [review]
